walkers/src/zoom.rs

```diff
@@ -9,9 +9,8 @@ impl TryFrom<f64> for Zoom {
     type Error = InvalidZoom;
 
     fn try_from(value: f64) -> Result<Self, Self::Error> {
-        // Mapnik supports zooms up to 19.
-        // https://wiki.openstreetmap.org/wiki/Slippy_map_tilenames#Zoom_levels
-        if !(0. ..=19.).contains(&value) {
+        // The upper limit is artificial. Should it be removed altogether?
```

Comment:
I think some limit needs to stay. I did not check it, but I imagine that some things like dragging might break, or start to misbehaving. Or maybe it should be configured. Either way, this can be left for later.